### ISSUE TITLE
Autofs doesn't mount the misc paths, /var/snd is mounted OK

### ISSUE THREAD

**Issue by vizubeat** [ElvishArtisan/ubuntu-rivendell-installer#1]
Apologies if this isn't the right repo but I've found some references to Autofs in this repo.

I've discovered that my 4.2.2 server/client setup only actually mounts `/var/snd` - the 'misc' entries get created in all the right places but the server's NFS exports don't get mounted on the client, even though the entries exist in `/etc/auto.misc`.

I don't know enough about Autofs, but I lifted these lines from `/etc/auto.misc` and added them to the bottom of `/etc/auto.rd.audiostore` (even though the file warns me not to edit the file by hand!):

```
/misc/rd_xfer           -fstype=nfs,rw,noexec   192.168.1.10:/home/rd/rd_xfer
/misc/music_import      -fstype=nfs,rw,noexec   192.168.1.10:/home/rd/music_import
/misc/music_export      -fstype=nfs,rw,noexec   192.168.1.10:/home/rd/music_export
/misc/traffic_import    -fstype=nfs,rw,noexec   192.168.1.10:/home/rd/traffic_import
/misc/traffic_export    -fstype=nfs,rw,noexec   192.168.1.10:/home/rd/traffic_export

```

...then running `sudo service autofs restart` does mount the `/misc/` items correctly and I can browse the directories.

So I'm not sure what's missing to make sure that the entries in `/etc/auto.misc` get picked up by Autofs.

Maybe there needs to be a new file added to `/etc/auto.master.d/` to refer to `/etc/auto.misc` to pick up the `/misc/` items?

(Side quest, just out of interest, why was Autofs chosen over adding entries to `/etc/fstab`?)

**Comment by MikeTheHammer** [comment]
I'm working on this. 

It seems there are two problems here. But, the real solution is to do things similar to what @vizubeat did. 

The first is that there is a line in auto.master that needs to be un-commented.
`#/misc	/etc/auto.misc`

With this line commented out, the auto.misc file is ignored.

The second problem is that, since the line above starts with `/misc` and not `/-` , the individual lines in auto.misc need to just start with the folder name, no leading `/misc/`. 

So, the lines would need to be 
`rd_xfer           -fstype=nfs,rw,noexec   192.168.1.10:/home/rd/rd_xfer`
etc.


However, from reading the tea leaves on various documentation sites for autofs, (particularly here [https://wiki.archlinux.org/title/Talk:Autofs](url) ), I think the preferred way to do this, especially with automated install scripts like this, is to create dedicated files for the mapping, as the installer already does for the audiostore.

I'm putting together a pull request to move these lines to a `auto.rd.xfer` file and add a `/etc/auto.master.d/rd.xfer.autofs` file with the contents:
`/-      /etc/auto.rd.xfer`

I've tested it with Jammy and it works. I'm going to test with Focal, Noble, and Mint Virginia next. I can't think of any reason those wouldn't work exactly the same, but you never know. 

I hope to have a pull request ready by end of Tuesday.

**Comment by MikeTheHammer** [comment]
It appears that Focal is no longer supported, the public keys weren't available. (Understandable since there's less than a year of support from Ubuntu left for Focal.)

I tested Jammy, Noble, and Mint Virginia. Everything worked.

**Comment by ElvishArtisan** [comment]
> It appears that Focal is no longer supported, the public keys weren't available. (Understandable since there's less than a year of support from Ubuntu left for Focal.)

Correct. Focal was never "officially" supported, although there were a few early Rivendell beta releases on it. Then Jammy came out before Rivendell v4.x came out of beta, so it was easier to just move everything forward to it.

**Comment by vizubeat** [comment]
Thanks @MikeTheHammer this looks great, hope we can get it merged! :-)
